### PR TITLE
Typos fixed and link updated in `mathml.py`

### LINKED ISSUE #11986: Typo Error in mathml.py
In the file [https://github.com/sympy/sympy/blob/master/sympy/printing/mathml.py](https://github.com/sympy/sympy/blob/master/sympy/printing/mathml.py), there is a typo:
 in line `106`,  `# thing a coeff of 1 can remain` , `thing` should be replaced by `think`

### PR DESCRIPTION
**Pull request by mamakancha**
Two typos have been fixed.
And the link given in `Line 21` has been updated.

This PR also fixes #11986

### CODE REVIEW

**Comment by mamakancha** [comment]
@jksuom Can you please restart the falling **Theano** test?

File: sympy/printing/mathml.py [hunk 1]
```diff
@@ -165,7 +165,7 @@ def _print_Rational(self, e):
         #numerator
         xnum = self.dom.createElement('cn')
         xnum.appendChild(self.dom.createTextNode(str(e.p)))
-        #denomenator
+        #denominator
```
Comment (aktech) [inline sympy/printing/mathml.py]: comment should start with a `#` and a "single space".

**Review by aktech — CHANGES_REQUESTED** [review]

File: sympy/printing/mathml.py (hunk 1 above)
Comment (mamakancha) [inline sympy/printing/mathml.py]: Thanks for pointing out! Done :)

**Comment by mamakancha** [comment]
`./bin/test test_refine.py` seems to take a lot of time and hence leads to termination of the test. What could be the possible reason for this due to the edits?

**Comment by mamakancha** [comment]
ping @aktech @jksuom

**Comment by smichr** [comment]
It's in. Thanks!

**Pull request merged**